app/services/jwt_token.rb

```diff
@@ -35,7 +35,11 @@ def to_s
 
   # This method does not verify if the token signature is valid
   def decoded_payload
-    @decoded_payload ||= JWT.decode(token, nil, false).first
+    return @decoded_payload if defined?(@decoded_payload)
+    @decoded_payload = JWT.decode(token, nil, false).first
```

Comment:
I'm not sure I understand, how is this different to the original version of this line?

Reply:
I also learned this recently, refer to https://jtway.co/how-to-memoize-false-and-nil-values-fe28b8ede9f8

Reply:
nice!

Reply:
Yep, you got to be careful with this. :-)

Reply:
I'm often not a fan of code comments because many are too obvious, but this is a good candidate for one because it's not obvious :)

Reply:
@ekohl  Sure let me do that.